spec/lib/jenkins_pipeline_builder/extensions/conditions_spec.rb

```diff
@@ -16,8 +16,8 @@
   end
 
   before :each do
-    builder = Nokogiri::XML::Builder.new { |xml| xml.conditions }
-    @n_xml = builder.doc
+    condition = Nokogiri::XML::Builder.new { |xml| xml.conditions }
```

Comment:
Found out you cannot do `@n_xml = Nokogiri::XML::Builder.new { |xml| xml.conditions }`, tests will fail gloriously. I believe it has to do with how Nokogiri builds XML which is why we have to reassign @n_xml every time before each test. I could be wrong, haven't looked that deep into it.